pkg/shared/logging.go

```diff
@@ -47,11 +51,21 @@ func LogInfo(
 		cr.Namespace,
 		cr.Name,
 	)
+
+	if eventReason != "" {
```

Comment:
For the various spots with this check, we should probably do `if eventReason != EventReasonNoEvent`.